<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DUTWindow</class>
 <widget class="QMainWindow" name="DUTWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/DUT.png</normaloff>:/icons/icons/DUT.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="TestsTreeView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTestSuite">
    <property name="title">
     <string>TestSuite</string>
    </property>
    <addaction name="actionAddTestSuite"/>
    <addaction name="actionRemoveTestSuite"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionStartRunner"/>
    <addaction name="actionPauseRunner"/>
   </widget>
   <addaction name="menuTestSuite"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="TaskQueueDock">
   <property name="windowTitle">
    <string>Task Queue</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QListView" name="taskQueueListView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddTestSuite"/>
   <addaction name="actionRemoveTestSuite"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionStartRunner"/>
   <addaction name="actionPauseRunner"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="TestInfoDock">
   <property name="windowTitle">
    <string>Test Info</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QTableView" name="testInfoTableView">
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAddTestSuite">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/addTestSuite.png</normaloff>:/icons/icons/addTestSuite.png</iconset>
   </property>
   <property name="text">
    <string>addTestSuite</string>
   </property>
  </action>
  <action name="actionRemoveTestSuite">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/removeTestSuite.png</normaloff>:/icons/icons/removeTestSuite.png</iconset>
   </property>
   <property name="text">
    <string>removeTestSuite</string>
   </property>
  </action>
  <action name="actionLockDUT">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/lock.png</normaloff>:/icons/icons/lock.png</iconset>
   </property>
   <property name="text">
    <string>lockDUT</string>
   </property>
  </action>
  <action name="actionReleaseDUT">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/releaseLock.png</normaloff>:/icons/icons/releaseLock.png</iconset>
   </property>
   <property name="text">
    <string>releaseDUT</string>
   </property>
  </action>
  <action name="actionStartRunner">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/run.png</normaloff>:/icons/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>StartRunner</string>
   </property>
   <property name="toolTip">
    <string>StartRunner</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/refresh.png</normaloff>:/icons/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionPauseRunner">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/pause.png</normaloff>:/icons/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>PauseRunner</string>
   </property>
   <property name="toolTip">
    <string>PauseRunner</string>
   </property>
  </action>
  <action name="actionAddtoTaskQueue">
   <property name="text">
    <string>addtoTaskQueue</string>
   </property>
   <property name="toolTip">
    <string>addtoTaskQueue</string>
   </property>
  </action>
  <action name="actionRemoveFromTaskQueue">
   <property name="text">
    <string>removeFromTaskQueue</string>
   </property>
   <property name="toolTip">
    <string>removeFromTaskQueue</string>
   </property>
  </action>
  <action name="actionClearTaskQueue">
   <property name="text">
    <string>clearTaskQueue</string>
   </property>
   <property name="toolTip">
    <string>clearTaskQueue</string>
   </property>
  </action>
  <action name="actionRemoveResult">
   <property name="text">
    <string>removeResult</string>
   </property>
   <property name="toolTip">
    <string>removeResult</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
